### PR TITLE
Support old restart policy in compose v3

### PR DESCRIPTION
**Pull request by hangyan**
related to #876

### CODE REVIEW

**Comment by surajnarwade** [comment]
LGTM

File: script/test/fixtures/restart-options/docker-compose-restart-unless-stopped.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+version: "3"
+
+services:
+  foo:
+    image: "foobar"
+    restart: "unless-stopped"
+    environment:
+      GITHUB: surajssd
+
```
Comment (surajnarwade) [inline script/test/fixtures/restart-options/docker-compose-restart-unless-stopped.yml]: remove blank line

Comment (hangyan) [reply script/test/fixtures/restart-options/docker-compose-restart-unless-stopped.yml]: @surajnarwade updated! thanks!

**Comment by surajnarwade** [comment]
@hangyan one last thing, can you please add documentation stating that we are converting `unless-stopped` restart policy to `always`

**Comment by hangyan** [comment]
@surajnarwade thanks, I will remeber this in the feature PRs.

File: pkg/loader/compose/v3.go [hunk 2]
```diff
@@ -285,10 +285,16 @@ func dockerComposeToKomposeMapping(composeObject *types.Config) (kobject.Kompose
...
 
-		// restart-policy:
+		// restart-policy: deploy.restart_policy.condition will rewrite restart option
+		// see: https://docs.docker.com/compose/compose-file/#restart_policy
+		serviceConfig.Restart = composeServiceConfig.Restart
 		if composeServiceConfig.Deploy.RestartPolicy != nil {
 			serviceConfig.Restart = composeServiceConfig.Deploy.RestartPolicy.Condition
 		}
+		if serviceConfig.Restart == "unless-stopped" {
+			log.Warnf("Restart policy 'unless-stopped  in service %s is not supported, convert it to 'always'", name)
```
Comment (cdrage) [inline pkg/loader/compose/v3.go]: missing a `'` around `unless-stopped`

Comment (hangyan) [reply pkg/loader/compose/v3.go]: LGTM

Comment (hangyan) [reply pkg/loader/compose/v3.go]: updated!

**Comment by cdrage** [comment]
Awesome! Tests pass and this looks great to me :+1: merging!

**Pull request merged**